### PR TITLE
Testing infrastructure that allows using HTML fixtures with examples

### PR DESCRIPTION
**Pull request by oyiptong**
The examples show:
1. how to use HTML fixtures
2. how to execute scripts within a test

### CODE REVIEW

**Comment by oyiptong** [comment]
@marcoscaceres, @emtwo, @sarracini: happy testing

**Comment by oyiptong** [comment]
also r?
We can delete the test fixtures later, in another issue

**Comment by marcoscaceres** [comment]
Nice! :dancer:

**Comment by marcoscaceres** [comment]
R+

File: test/script_test_example.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  it("should be able to run scripts", () => {
+    document.body.innerHTML = __html__["test/fixtures/script_test_example.html"];
+    var elem = document.querySelector("#element-exists");
+    expect(elem).to.be.ok;
+    expect(elem.textContent).to.equal("");
+
+    var script = document.createElement("script");
+    script.setAttribute("type", "text/javascript;version=1.8");
+    script.textContent = "let elem = document.querySelector(\"#element-exists\"); elem.textContent = \"Hello World\";";
```
Comment (marcoscaceres) [inline test/script_test_example.js]: use "`"

**Pull request merged**